qiskit/providers/models/backendconfiguration.py

```diff
@@ -454,6 +461,13 @@ def __init__(self,
         self.dt = dt * 1e-9  # pylint: disable=invalid-name
         self.dtm = dtm * 1e-9
 
+        if channels is not None:
+            self.channels = channels
```

Comment:
Should this be private? It's unparsed data, right?

@taalexander

Reply:
Yes, I think so.

Reply:
Alright. Leaving this as public means we are committing to the schema as is (or have to go through deprecation to change it)

Reply:
No worries! Thank you @lcapelluto and @taalexander. 😊

Fixed in [6e4111a](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/6e4111afdde8e0a008a3884850cfc1580924e3b5).